bot/src/wrapper/_communication.py

```diff
@@ -208,3 +211,60 @@ async def caller(session: aiohttp.ClientSession) -> ShopBuyOutput:
                 raise UnknownNetworkError(message)
 
         return await make_request(session, caller)
+
+
+class UserRawAPI:
+    """A bundle of API available to use for user endpoint."""
+
+    @staticmethod
+    async def get_user(session: aiohttp.ClientSession, user_id: int) -> UserIdGetOutput:
+        """Get the User information by user id through direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> UserIdGetOutput:
+            async with session.get(f"user/{user_id}") as resp:
+                if resp.ok:
+                    return await resp.json()
+                if resp.status == Status.NOT_FOUND:
+                    message = f"User with user id {user_id} cannot be found"
+                    raise DoNotExistError(message)
+                message = f"Undefined behaviour bot.src.wrapper.UserRawAPI.get_user, Status received {resp.status}"
+                raise UnknownNetworkError(message)
+
+        return await make_request(session, caller)
+
+    @staticmethod
+    async def create_user(session: aiohttp.ClientSession, user_id: int) -> None:
+        """Create the user by user id through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> None:
+            async with session.post(f"user/{user_id}") as resp:
```

Comment:
Technically, `user/{user_id}`, when looking at it, says `get user with this user_id`. Thus, making a post request to `user/{user_id}` to create a new user is not ideal. The endpoint itself, with a quick glance at it, show tell the reader everything they need to know about its function.

So, a POST to `/user` is good, even `/user/create`.

Reply:
Not quite sure what you meant by this, you mean the API? That was also from v2

Reply:
Would be the API endpoint. Forgot this is the wrapper. I have not seen anything for the `user` endpoints, but my statement would apply more to when those endpoints are available.

But in summary, creating a user with POST `/user` is the ideal method.

Reply:
Alright, maybe you could change the reference in v3 with the schema and params? I am not quite sure of the correct practice of this but I do know it is not too ideal.